docs/workflow.md

```diff
@@ -403,3 +392,288 @@ workflow {
 ```
 
 In the above snippet, the initial channel is piped to the {ref}`operator-map` operator, which reverses the string value. Then, the result is passed to the processes `foo` and `bar`, which are executed in parallel. Each process outputs a channel, and the two channels are combined using the {ref}`operator-mix` operator. Finally, the result is printed using the {ref}`operator-view` operator.
+
+(workflow-output-def)=
+
+## Publishing outputs
```

Comment:
I think the core concept should be "Workflow output definition" which essentially allows structuring the output of a workflow execution and collect publishing targets

Reply:
I hesitate to call it "workflow output definition" here because there is already a section called "workflow outputs (emit)". Even with the "emit" qualification, it still feels confusing to have two different sections about workflow "outputs". That is why I focus on "publishing" here to better distinguish from emits.

Reply:
Ok, for now. We may want to reshape it a bit when promoting it stable